src/pulse3D/excel_writer.py

```diff
@@ -181,6 +181,7 @@ def create_frequency_vs_time_charts(
 
 def write_xlsx(
     plate_recording: PlateRecording,
+    disable_y_normalization: bool = False,
```

Comment:
IMO when designing APIs in general you typically want to avoid using negative-prefixed words like `disable`. You can see here we have a double negative on this line now: by default we are `not` `dis`abling y-axis normalization, or in other words we are simply enabling y-axis normalization. This kwarg should probably just be `normalize_y_axis` and be set to True by default